versions/3.0.md

```diff
@@ -1152,6 +1152,51 @@ name: pet
 description: Pets operations
 ```
 
+#### <a name="examplesObject"></a>Examples Object
+
+Anywhere an `example` may be given, allow a $ref object.  This does mean that `example` can be either a string primitive or an object, like `additionalProperties`:
```

Comment:
@ePaul - when writing the spec, while making mistakes here and there, I've taken great care to word things as they are. Type `Any` means just that - any valid JSON value would fit there, regardless of what it relates to. The word `SHOULD` is used on purpose and not the word `MUST`. The spec really does not enforce any sort of constraint here, and I don't think it should (for a few reason).

Going back to the `default` field value, you'll see that it uses a stronger restriction saying it `MUST` follow the type assigned to its container.